src/Packages/Passport/Runtime/ThirdParty/Gree/Assets/Plugins/WebGLTemplates/unity-webview/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Unity WebGL Player | Immutable Runner</title>
```

Comment:
Should it say Immutable Runner?
Also, could we remove any other stuff from <head>? The same applies to other files.